scripts/standard_configure.sh

```diff
@@ -45,4 +45,4 @@ rm -rf "${KRATOS_BUILD}/${KRATOS_BUILD_TYPE}/CMakeFiles"
 cmake -H"${KRATOS_SOURCE}" -B"${KRATOS_BUILD}/${KRATOS_BUILD_TYPE}" -DUSE_MPI=OFF
 
 # Buid
-cmake --build "${KRATOS_BUILD}/${KRATOS_BUILD_TYPE}" --target install -- -j4
+cmake --build "${KRATOS_BUILD}/${KRATOS_BUILD_TYPE}" --target install -- -j$(nproc)
```

Comment:
`nproc` is an env-var in unix?

Reply:
not defined on my machine :eyes:

Reply:
It is a command, and should be unix standard. the problem is I don't think everyone will get what they expect. For what I've been able to read, this returns the number of procs available to the process calling it. For example, my computer limits the number of cpu that a terminal can use to one, so whenever I use it it always fails to 1 (Even when I have 4).

I suppose that's the reason I had id hardcoded to 4 when I uploaded it in the first place.